usaspending_api/etl/award_helpers.py

```diff
@@ -33,6 +33,14 @@ def update_awards(award_tuple: Optional[tuple] = None) -> int:
     award records would be an ETL bottleneck.
     """
 
+    _prune_empty_awards_cte = str(
```

Comment:
This isn't a CTE. 

I looked at the existing FPDS loader and I see there is more than just deleting the record from `awards`. Since this is used as a FK, other tables need to be updated to replace the soon-to-be-deleted award ids with nulls. Perhaps `update_awards()` should call another function to handle the deletes. That function can also handle updating the other tables.

IF the SQL stays in this file, the SQL strings should match the other SQL in the file to maintain consistency.

- alias `transaction_normalized` as "tn"
- split the SQL sub-query across multiple lines with two-space indentations
- capitalize "is null" to "IS NULL"

Reply:
Only part of the requested changes were made